server/Assignments_filler/sheetFiller.py

```diff
@@ -0,0 +1,216 @@
+import os
+import logging
+import sys
+import time
+
+import DB_call
+from DB_call import get_from_db
```

Comment:
`DB_call` imports in the third "import group". You also don't need standalone `DB_call`.

Reply:
Scratch that, you do, because you start the connection that way. A possibly better way to do this would be to have an `init` call in `DB_call.py`, that you run once in `sheetFiller.py` before starting the `checker`. Then you can just import the 2 functions you use and keep it nicely separated.

Reply:
I don't understand that one.
Do you mean I don't need line 7 since line 6 already covers it?
if you do - I know, just makes it a bit shorter overall.

Reply:
No, I mean you shouldn't need line 6. :)